templates/zerver/faq.html

```diff
@@ -102,6 +102,32 @@ <h1>Frequently asked questions</h1>
                     open source project or similar organization.
                 </p>
             </div>
+            <div class="faq">
+                <div class="question">
+                    What is the difference between automatic and manual billing?
+                </div>
+                <p class="answer">
+                    With automatic billing, you only pay for the current number of
+                    users in your organization. With manual billing, you choose a
+                    preset user limit. If the limit is reached, no more users can
+                    join until additional licenses are paid for. In general, we
+                    highly recommend selecting automatic billing unless you have
+                    a specific reason to do otherwise.
+                </p>
+            </div>
+            <div class="faq">
+                <div class="question">
+                    How will I be charged for temporary users (e.g. limited-time
+                    clients)?
+                </div>
+                <p class="answer">
+                    With automatic license management, you stop paying for a user
```

Comment:
They way licenses work is similiar for both automatic and manual license management. To add a new user you need atleast one spare license. And when the user is deactivated the license is freed and can be used for addning a new user. So once a license is purchased by the organization, it remains valid till the end of the billing period iresepctive of whether its fully used or not. This is true for both manual and automatic license managment.

The diffrence between manual and automatic license management
* When a new user tries to sign up and there are no more spare licenses,
  * In case of manual the sign up fails.
  * In case of automatic, the system automatically purchases a new license on behlf of the user.
* How the number of licenses is calculated for the next billing period
  *   For automatic, this would be equal to the number of activated users in the organization at the end of the current billing period.
  * For manual, this quantity is controlled by the users and can be adjusted in /billing page. Though with condition that the number of licenses in next billing period should be atleast the number of activated users. Also for charge by invoice there is also a min 30 licenses requirement.

Reply:
This is very helpful! I'll copy these comments over to a more permanent location and update the PR next week.